test/integration/crud/find_cursor_methods.test.js

```diff
@@ -204,7 +204,8 @@ describe('Find Cursor', function () {
     });
   });
 
-  context('#clone', function () {
+  // TODO(NODE-4988): Need to fix cursor.clone in mongodb-legacy
+  context.skip('#clone', function () {
```

Comment:
since it's just the two tests, could we fix just fix the tests and not have the skip?

Reply:
`async` -ified and fixed up 👍